src/form/wrm.jl

```diff
@@ -76,19 +93,340 @@ function variable_voltage(pm::GenericPowerModel{T}; nw::Int=pm.cnw, cnd::Int=pm.
         var(pm, nw, cnd, :wr)[(i,j)] = WR[w_fr_index, w_to_index]
         var(pm, nw, cnd, :wi)[(i,j)] = WI[w_fr_index, w_to_index]
     end
-
 end
 
+function variable_voltage(pm::GenericPowerModel{SDPDecompForm}; nw::Int=pm.cnw, cnd::Int=pm.ccnd, bounded=true)
```

Comment:
This function signature should be,
```
function variable_voltage(pm::GenericPowerModel{T}; nw::Int=pm.cnw, cnd::Int=pm.ccnd, bounded = true) where T <: SDPDecompForm
```
Otherwise it cannot be extended later on.